sormas-rest/src/main/java/de/symeda/sormas/rest/ExternalVisitsResource.java

```diff
@@ -30,6 +35,23 @@ public Boolean isValidPersonUuid(@PathParam("personUuid") String personUuid) {
 		return FacadeProvider.getPersonFacade().isValidPersonUuid(personUuid);
 	}
 
+	@POST
+	@Path("/person/{personUuid}/status")
+	public boolean postSymptomJournalStatus(@PathParam("personUuid") String personUuid, Map<String, Object> body) {
+		try {
+			PersonDto person = FacadeProvider.getPersonFacade().getPersonByUuid(personUuid);
+			SymptomJournalStatus statusValue = SymptomJournalStatus.valueOf(body.get("status").toString());
+			person.setSymptomJournalStatus(statusValue);
+			FacadeProvider.getPersonFacade().savePerson(person);
+			return true;
+		} catch (Exception e) {
+			Logger logger = LoggerFactory.getLogger(ExternalVisitsResource.class);
```

Comment:
in the parent class there is a logger defined, please use that one

Reply:
As you suggested, I have moved the logic from the ExternalVisitRessource to the PersonFacade. The Exception is now thrown from in there, and automatically logged. The ExternalVisitRessource appears in the stack trace. Is that also acceptable?
When I forward the exception and log it from within the ExternalVisitRessource with the parent's logger, the server log looks messier...